addons/sale/static/src/js/product_catalog/sale_order_line/sale_order_line.scss

```diff
@@ -10,8 +10,7 @@ div.o_sale_product_catalog_quantity {
     }
 }
 
-// Keeping this here until rd-design make a global class for this color.
-button.o_sale_product_catalog_add_to_cart {
-    background-color: #017e84;
-    color: RGBA(255, 255, 255, var(--text-opacity, 1));
+.o_sale_product_kanban_view .o_kanban_renderer .o_kanban_record .o_sale_product_added {
```

Comment:
Is it mandatory to keep this long path ?
Can we use this ?
```css
.o_sale_product_added { }
```

Reply:
Without this specificity in the selector, the bg and border are not applied because of another more specific selector applying a white bg and grey border (`.o_kanban_quick_create`)